tests/beaker_tests/cisco_vlan/vlan/vlanlib.rb

```diff
@@ -89,7 +89,7 @@ def self.create_stdvlan_manifest_absent
   # vlan_name, state and shutdown.
   # @param none [None] No input parameters exist.
   # @result none [None] Returns no object.
-  def self.create_stdvlan_manifest_nondefaults(test_mapped_vni)
+  def self.create_stdvlan_manifest_nondefaults(test_mapped_vni, test_fabric_control)
```

Comment:
Instead of extending this method to use additional arguments, I think it would be better to use a hash.
```Ruby
def self.create_stdvlan_manifest_nondefaults(test_properties)`
...
if test_properties[:fabric_control]
  ...
```
**Usage**
```Ruby
test_properties = {
  mapped_vni:       platform.match('n9k')
  fabric_control:   platform.match('n7k')
}
```